### PR TITLE
Fix layer thresholds popup

### PR DESCRIPTION
**Pull request by Tomaz-Vieira**
Simplifies NormalizableLayer, removing the nebulous concept of "range" in favor of only using the "normalize" values. Cleans up and removes a lot of replication from layerDialog.py

Companion ilastik PR: https://github.com/ilastik/ilastik/pull/2282

Fixes #https://github.com/ilastik/ilastik/issues/2281
Fixes https://github.com/ilastik/ilastik/issues/2195

### CODE REVIEW

File: volumina/widgets/layerDialog.py [hunk 1]
```diff
@@ -27,146 +27,78 @@
...
 
-    def setLayername(self, n):
-        self._layerLabel.setText("<b>%s</b>" % n)
+class RGBALayerDialog(LayerDialog):
+    def __init__(self, layer, parent=None):
+        super().__init__(ui_file_name="rgbaLayerDialog.ui", layer=layer, parent=parent)
+
+        for idx, (t_widget, autorange_checkbox, channel) in enumerate(
+            [
+                (self.redChannelThresholdingWidget, self.redAutoRange, self.redChannel),
```
Comment (k-dominik) [inline volumina/widgets/layerDialog.py:89]: I cannot find these functions `redAutoRange`, `greenAutoRange`... anywhere

Comment (k-dominik) [reply volumina/widgets/layerDialog.py:89]: fascinatingly enough it works - no errors related to this.

Comment (Tomaz-Vieira) [reply volumina/widgets/layerDialog.py:93]: This is the reason I have trust issues with IDE's; they will sometimes miss things that `grep`-like things won't ^^

`redAutoRange`, `greenAutoRange`, etc are checkbox widgets defined at `volumina/widgets/ui/rgbaLayerDialog.ui` and `volumina/widgets/ui/grayLayerDialog.ui` =P

File: volumina/widgets/layerDialog.py [hunk 2]
```diff
@@ -27,146 +27,78 @@
...
-        self.grayChannelThresholdingWidget.valueChanged.connect(dbgPrint)
-        self.grayAutoRange.stateChanged.connect(autoRange)
-        self.grayAutoRange.setCheckState(layer._autoMinMax[0] * 2)
-
-    def setLayername(self, n):
-        self._layerLabel.setText("<b>%s</b>" % n)
+    def initialize_range_widgets(self, datasourceIdx: int, thresholding_widget: ThresholdingWidget, autorange_checkbox):
+        def handleRangeChanged(a, b):
+            self.layer.set_normalize(datasourceIdx, (a, b))
+            print(f"normalization  aeeee changed for channel={datasourceIdx} to [{a}, {b}]")
```
[the remark this answers is not in the document]
Comment (Tomaz-Vieira) [reply volumina/widgets/layerDialog.py]: oops thanks

File: volumina/widgets/layerDialog.py (hunk 1 above)
Comment (k-dominik) [reply volumina/widgets/layerDialog.py:93]: ah yeah, I was grepping, too, but only in *.py files :) my miss.!

File: volumina/widgets/layerDialog.py [hunk 3]
```diff
@@ -27,146 +27,82 @@
...
         self.setLayername(layer.name)
+        self.layer = layer
 
-        def dbgPrint(a, b):
-            layer.set_normalize(0, (a, b))
-            logger.debug("normalization changed to [%d, %d]" % (a, b))
+    def initialize_range_widgets(self, datasourceIdx: int, thresholding_widget: ThresholdingWidget, autorange_checkbox):
+        def handleRangeChanged(a, b):
+            self.layer.set_normalize(datasourceIdx, (a, b))
+            print(f"normalization changed for channel={datasourceIdx} to [{a}, {b}]")
```
Comment (k-dominik) [inline volumina/widgets/layerDialog.py:54]: hang on, I would prefer to either have this a debug log message, or gone completely

**Comment by k-dominik** [comment]
perfect!

**Pull request merged**